src/compressed_tensors/utils/safetensors_load.py

```diff
@@ -238,3 +256,17 @@ def is_quantization_param(name: str) -> bool:
         return True
 
     return False
+
+
+def get_nested_mappings_from_state_dict(state_dict, params_to_nest):
```

Comment:
This function is not used?

Reply:
My bad, missed it during rebase, good catch!